test-resources/java/spark_conf/configurations/rules.toml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2023 Uber Technologies, Inc.
+# 
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+# 
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+name = "spark_conf_change_java_scala"
+query = "cs new SparkConf()"
+replace_node = "*"
+replace = 'new SparkSession.builder()'
+
+
```

Comment:
Minor nit: but let's make the spacing between rules consistently, either one or two empty lines, but not a mix of the two unless it implies a particular grouping.